guides/common/modules/proc_adding-custom-deb-repositories.adoc

```diff
@@ -0,0 +1,72 @@
+. Optional: In the *Description* field, enter a description for the repository.
+. From the *Type* list, select `deb` as type of repository.
+. In the *Upstream URL* field, enter the URL of the external repository to use as a source.
+You can find the upstream URLs on Debian-based systems in `/etc/apt/sources.list`.
+. Optional: In the *Releases* field, set a or multiple releases.
++
+For official Debian repositories, {Team} recommends setting a _codename_ in the *Releases* field, for example `buster` for Debian 10 or `bullseye` for Debian 11.
+Do not use `stable` or `testing` as they merely reference a currently corresponding _codename_.
+This helps to avoid drastic changes once a new Debian version is released and the reference is changed.
+{Team} recommends creating one repository in {Project} per release to keep things tidy and avoid potential performance and network issues during synchronization.
+For official Ubuntu repositories, use the Ubuntu suite, for example `focal` or `focal-updates`.
+. Optional: In the *Components* field, enter a component.
+This indicates the licensing terms of the software packages.
++
+In Debian, it is divided into `main`, `contrib`, and `non-free`.
+For official Debian or Ubuntu repositories, {Team} recommends leaving this field empty to synchronize all available components.
+Note that some third party Debian repositories use the components in ways that may require setting an explicit selection.
++
+[CAUTION]
+====
+Ensure to enter both *Releases* and *Components* exactly as they would be in an `/etc/apt/sources.list` file.
+====
+. Optional: In the *Architectures* field, enter one or multiple architectures to limit the synchronization of packages to a subset of the available architectures.
+Leave this field empty to synchronize all available architectures.
+ifdef::orcharhino[]
+. Optional: In the *Errata URL* field, enter the URL of an errata service.
+endif::[]
+. Select the *Verify SSL* check box if you want to verify that the upstream repository's SSL certificates are signed by a trusted CA.
+. In the *Upstream Username* field, enter the user name for the upstream repository if required for authentication.
+Clear this field if the repository does not require authentication.
+. In the *Upstream Password* field, enter the corresponding password for the upstream repository.
+Clear this field if the repository does not require authentication.
+. Ensure to select the *Mirror on Sync* check box.
+This ensures that the content that is no longer part of the upstream repository is removed during synchronization.
+. Optional: In the *HTTP Proxy Policy* field, select an HTTP proxy.
+. Optional: If you want, you can clear the *Publish via HTTP* check box to disable this repository from publishing through HTTP.
+. Optional: From the *GPG Key* list, select the GPG key if you wish to verify the signatures of the `Release` files associated with the Debian repository.
```

Comment:
Best to keep wishes for birthday candles.
```suggestion
. Optional: From the *GPG Key* list, select the GPG key if you want to verify the signatures of the `Release` files associated with the Debian repository.
```

Reply:
🕯️